projects/ngx-feature-flag/src/lib/service/enable-feature.service.ts

```diff
@@ -0,0 +1,92 @@
+import { HttpClient, HttpErrorResponse, HttpHeaders } from '@angular/common/http';
```

Comment:
@divyanshiGupta I guess this is a duplicate service from `fabric8-ui`. I am thinking if there is a way to reuse the same service here but because of cyclic dependency it would not be easy. We could have it done the way `PipelineService` is done in `ngx-launcher` but thats not the best way for me as well. I think this will be solved by the SDK refactoring.

Reply:
@rohitkrai03 this is a duplicate service because at this point I didnt find any better and less time consuming way of doing this. You are right about the cyclic dependency issue and hence I tried to do it by creating the service interface in ngx-feature-flag and implementing it in fabric8-ui the way PipelineService is done in ngx-launcher but this also didnt worked out for me as some other issues were coming up. I agree with you that this can be solved once we get all the services in the SDK and we can use this service both in fabric8-ui and ngx-feature-flag from the SDK.

Reply:
@divyanshiGupta Understood. I'll add this use case in SDK doc.